pkg/cmd/grafana-server/main.go

```diff
@@ -143,6 +144,9 @@ func executeServer(configFile, homePath, pidFile, packaging string, traceDiagnos
 	setting.BuildBranch = buildBranch
 	setting.IsEnterprise = extensions.IsEnterprise
 	setting.Packaging = validPackaging(packaging)
+	if len(distributor) >= 100 {
```

Comment:
Better to truncate?

Reply:
It has to have a length greater than 100? Why?

Reply:
fixed